changelogs/0.13.md

```diff
@@ -0,0 +1,87 @@
+# [0.13.0]
+
+## Added
+- Bitcoin sidechain
+    - Operation `sidechain_btc_approve_aggregate_operation`
+    - Method `send_withdraw_op` to `btc_sidechain`
+    - Fields `is_sent`, `echo_block_number` to `btc_withdraw_object`
+    - Field `is_relevant` to `btc_address_object`
+    - Field `remove_btc_address` signal to `database`
+- Fields `reciever`, `amount` to [block_reward_operation](/api-reference/echo-operations/block-reward.md#block_reward_operation)
+- Field `x86_64_maximum_contract_size` to `chain_parameters`
+- Database API
+    - New argument `type` (`` or `eth` or `btc`) for [get_account_withdrawals](/api-reference/echo-node-api/database-api.md#get_account_withdrawals-account), [get_account_deposits](/api-reference/echo-node-api/database-api.md#get_account_deposits-account)
+    - Method `get_block_rewards`
+- Added tests for x86_64 call of contract using `chain_interface`
+
+## Changed
+- Bitcoin sidechain
+	- Field `committee_member_ids_in_script` in `btc_address_object`, `btc_intermediate_deposit_object`, `btc_aggregating_object` is `map<account_id_type, echo::sidechain::btc::public_key>` now
+	- Rename method `get_active_committee_set` to `get_active_committee_pub_keys_map`
+- Field `task` in `registration_api` is `optional<registration_task>` now
+- Wallet API
+    - Rename `get_btc_addresses` to [get_btc_addresses](/api-reference/echo-wallet-api/README.md#get_btc_addresses-account)
+- Now frozen balances are taken into account, changed `get_balance` in `interval_tree_selector_impl`
+- Maximum number active committee members from `1001` to `19`
+- Database API
+    - Method `get_account_deposits` return type `variants` now
+    - Method `get_account_withdrawals` return type `variants` now`
+    - Rename `get_btc_addresses` to `get_btc_address`
+
+## Removed
+- Bitcoin sidechain
+	- Field `block_number` from `btc_withdraw_object`
+	- Object `compare_intermediate_by_ready_to_send`
+- Ethereum sidechain
+    - Field `block_num` in `pending_collect`
+- Field `rewards` in `block_reward_operation`
+- Field `next_available_vote_id` from `global_property_object`
+- Mechanism increase registration fee
+- Field `maximum_committee_count` from `chain_parameters`
+- Field `committee_member` from `committee_frozen_balance_deposit_operation`
+- Field `owner_approvals_to_remove` from `proposal_update_operation`
+- Fields `owner_approvals_to_add`, `owner_approvals_to_remove` from `approval_delta`
+
+## Fixed
+- Fallback signatures verification
+- Issue block_reward_operation for every account
```

Comment:
Block reward are accumulated and issues with block_reward_operation during maintenance